client/src/services/friend/friend.service.js

```diff
@@ -0,0 +1,40 @@
+export default class FriendService {
+  constructor($http) {
+    this.$inject = ['$http'];
+    this.friends = [];
+    this.friendRequests = [];
+  }
+
+  loadFriends() {
+    return this.$http.get('/api/friends').then(friends => {
+      friends.forEach(friend => {
+        if (friend.status === 'Accepted') {
```

Comment:
I suspect string matching will be simpler with all lower case. capitalizing seems to add unnecessary complexity.